impl/maven-impl/src/test/java/org/apache/maven/impl/DefaultModelXmlFactoryTest.java

```diff
@@ -19,13 +19,17 @@
 package org.apache.maven.impl;
 
 import java.io.StringReader;
+import java.io.StringWriter;
 
+import org.apache.maven.api.model.InputLocation;
 import org.apache.maven.api.model.Model;
 import org.apache.maven.api.services.xml.XmlReaderException;
 import org.apache.maven.api.services.xml.XmlReaderRequest;
+import org.apache.maven.api.services.xml.XmlWriterRequest;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
+import static org.assertj.core.api.Assertions.assertThat;
```

Comment:
please use standard JUnit asserts only. I'd like to keep third party dependencies to a minimum

Reply:
For the sake of this PR I will make it use `assertEquals` from JUnit. 

I do however think that there should be a statement from the (core) team to either:

- Remove the dependency entirely so it can not be used in the first place.
- Agree to keep the dependency and accept the use of it.

This distracts from the feedback on actual change, and will be a recurring discussion on other PR's.

I'm happy to open a thread to discuss this on the mailing list. WDYT?